### ISSUE TITLE
The Layer has never been called and thus has no defined output shape.

### ISSUE THREAD

**Issue by aberger7** [penny4860/YOLO-detector#40]
![Capture](https://user-images.githubusercontent.com/8903580/87347904-ed0f0a80-c521-11ea-83ee-c2315dbc21d4.PNG)

`{
    "model" : {
        "architecture":         "ResNet50",
        "input_size":           416,
        "anchors":              [0.57273, 0.677385, 1.87446, 2.06253, 3.33843, 5.47434, 7.88282, 3.52778, 9.77052, 9.16828],
        "labels":               ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"],
        "coord_scale" : 		1.0,
        "class_scale" : 		1.0,
        "object_scale" : 		5.0,
        "no_object_scale" : 	1.0
    },
    "pretrained" : {
        "full":   				"svhn/weights.h5"
    },
    "train" : {
        "actual_epoch":         20,
        "train_image_folder":   "tests/dataset/svhn/imgs/",
        "train_annot_folder":   "tests/dataset/svhn/anns/",
        "train_times":          5,
        "valid_image_folder":   "tests/dataset/svhn/imgs/",
        "valid_annot_folder":   "tests/dataset/svhn/anns/",
        "valid_times":          1,
        "batch_size":           2,
        "learning_rate":        1e-4,
        "saved_folder":   		"svhn",
        "jitter":				false,
        "first_trainable_layer": "input_1",
        "is_only_detect" : 		true
    }
}
`
Python: 3.7
TF: 2.3.0rc1
Keras: 2.4.3
OpenCV: 3.4.2.16

Don't have any idea what this means or why it's happening. If anyone can help it would be greatly appreciated.

**Comment by Shivankit99** [comment]
@aberger7 How did you solve this im facing the same issue

**Comment by ghost** [comment]
This can be simply fixed by using TF 1.15.0. I thought I was using it, but actually I was using TF 2.3.X.
Had to install the package with conda install tensorflow=1.15.0 and run the python file with "python test.py" instead of "py test.py", because on my machine python is for conda and py for python.

Regardless of it running now, the pre-trained model is no good. It detects numbers everywhere in the test pictures even if the threshold is set high.
